### DISCUSSION TITLE
Replace Phone Numbers With Contact Names

### DISCUSSION THREAD

**Discussion by zoelee70** [ReagentX/imessage-exporter#167] [Q&A] [answered]
Thank you for this useful code.  My iCloud storage is full because of messages and I have been deleting old messages after screen printing ones I want to save.  The print option in iMessage crashes when messages are multiple years old.  My plan is to backup all my messages using your exporter and then change settings so messages are only kept for a year.  

I am very new to programming but was able to load your program using home-brew and it worked with no problems.  Is it possible to change the output files so that the phone numbers are replaced with the contact names?

**Comment by ReagentX** [comment, marked as answer]
See #62
